apps/src/sites/studio/pages/projects/index.js

```diff
@@ -3,123 +3,52 @@ import React from 'react';
+  setPersonalProjects,
+  setPublicProjects
 } from '@cdo/apps/templates/projects/projectsRedux';
 import publishDialogReducer from '@cdo/apps/templates/projects/publishDialog/publishDialogRedux';
 import deleteDialogReducer from '@cdo/apps/templates/projects/deleteDialog/deleteProjectDialogRedux';
 
 $(document).ready(() => {
-  const script = document.querySelector('script[data-projects]');
-  const projectsData = JSON.parse(script.dataset.projects);
-
+  const projectsData = getScriptData('projects');
   registerReducers({
     projects,
     publishDialog: publishDialogReducer,
     deleteDialog: deleteDialogReducer
   });
   const store = getStore();
-  setupReduxSubscribers(store);
-  ReactDOM.render(
-    <Provider store={store}>
-      <GallerySwitcher />
-    </Provider>,
-    document.getElementById('gallery-navigation')
-  );
 
-  ReactDOM.render(
-    <Provider store={store}>
-      <ProjectHeader
-        canViewAdvancedTools={projectsData.canViewAdvancedTools}
-        projectCount={projectsData.projectCount}
-      />
-    </Provider>,
-    document.getElementById('projects-header')
-  );
-
-  const isPublic = window.location.pathname.startsWith('/projects/public');
-  const initialState = isPublic ? Galleries.PUBLIC : Galleries.PRIVATE;
+  const initialState = projectsData.isPublic
+    ? Galleries.PUBLIC
+    : Galleries.PRIVATE;
   store.dispatch(selectGallery(initialState));
-  const url = `/api/v1/projects/gallery/public/all/${MAX_PROJECTS_PER_CATEGORY}`;
-
-  $.ajax({
-    method: 'GET',
-    url: url,
-    dataType: 'json'
-  }).done(projectLists => {
-    store.dispatch(setProjectLists(projectLists));
-    const publicGallery = document.getElementById('public-gallery');
-    ReactDOM.render(
-      <Provider store={store}>
-        <PublicGallery limitedGallery={projectsData.limitedGallery} />
-      </Provider>,
-      publicGallery
-    );
-  });
-
-  const personalProjectsUrl = `/api/v1/projects/personal`;
-
-  $.ajax({
-    method: 'GET',
-    url: personalProjectsUrl,
-    dataType: 'json'
-  }).done(personalProjectsList => {
-    store.dispatch(setPersonalProjectsList(personalProjectsList));
-    ReactDOM.render(
-      <Provider store={store}>
-        <PersonalProjectsTable canShare={projectsData.canShare} />
-      </Provider>,
-      document.getElementById('react-personal-projects')
-    );
-  });
-
-  const publishConfirm = document.getElementById('publish-confirm');
+  store.dispatch(setPersonalProjects());
+  store.dispatch(setPublicProjects());
 
   ReactDOM.render(
     <Provider store={store}>
-      <PublishDialog />
+      <div>
+        <ProjectHeader
+          canViewAdvancedTools={projectsData.canViewAdvancedTools}
+          projectCount={projectsData.projectCount}
+        />
+        <GallerySwitcher
+          limitedGallery={projectsData.limitedGallery}
+          canShare={projectsData.canShare}
+        />
+        {/* TODO: Move components below into <PersonalProjectsTable/>? */}
```

Comment:
Those Dialogs are only relevant from the Personal Project Table, so sounds good to me!

Reply:
lol i almost forgot about this! i'll make a follow-up PR to scooch these over 🎉